src/docs/examples/formioV2.html

```diff
@@ -0,0 +1,6 @@
+<head>
+  <meta http-equiv="refresh" content="2; URL=https://www.bitdegree.org/" />
+</head>
+<body>
+<p>If you are not redirected in 2 seconds, <a href="https://www.bitdegree.org/">click here</a>.</p>
```

Comment:
Whats this? this does not look right

Reply:
reverted the content, some test content got committed. Those are just examples for internal testing.

Reply:
> Whats this? this does not look right

I deleted those examples of specific forms as those might cause some issues if those are not live and in the examples folder if someone finds those.